<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard</class>
 <widget class="QDialog" name="Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dashboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="src.qrc">
    <normaloff>:/images/ghjk.png</normaloff>:/images/ghjk.png</iconset>
  </property>
  <widget class="QPushButton" name="AddIncome">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>280</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Add Income Sources</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddExoenses">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Add Expenses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetBudgets">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>280</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Set Budgets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MonthlyReports">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>280</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Monthly Reports</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExpensesStatus">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Current Status of Expenses</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddLectureShedule">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>420</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Add Lecture Shedule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddImportantDates">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>420</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Add Important Dates</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DisplayLec">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>420</y>
     <width>241</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka Banner</family>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(122, 1, 142, 255));
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
border:1;</string>
   </property>
   <property name="text">
    <string>Display Lecture Shedule</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>70</y>
     <width>1361</width>
     <height>971</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/6071541.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>9</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">image: url(:/images/ghjk.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Sitka Banner</family>
        <pointsize>31</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(23, 0, 27);</string>
      </property>
      <property name="text">
       <string>Dashboard</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="Alert">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>40</y>
     <width>391</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>AddIncome</zorder>
  <zorder>AddExoenses</zorder>
  <zorder>SetBudgets</zorder>
  <zorder>MonthlyReports</zorder>
  <zorder>ExpensesStatus</zorder>
  <zorder>AddLectureShedule</zorder>
  <zorder>AddImportantDates</zorder>
  <zorder>DisplayLec</zorder>
  <zorder>Alert</zorder>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
